### PR TITLE
fix: downgrade blobfuse to 1.1.1 since there is breaking change on 1.2.3

### PR DESCRIPTION
**Pull request by andyzhangx**
fix: downgrade blobfuse to 1.1.1 since there is breaking change on 1.2.3
more details could be found here: https://github.com/Azure/kubernetes-volume-drivers/issues/66#issuecomment-649188681

@xuto2

### CODE REVIEW

**Comment by andyzhangx** [comment]
BTW, there is another `make` issue (not related to this PR):
```
# make generated
make: *** No rule to make target 'generated'.  Stop.
root@andydev:~/go/src/github.com/Azure/AgentBaker# make
make -C hack/tools/
make[1]: Entering directory '/root/go/src/github.com/Azure/AgentBaker/hack/tools'
GOBIN=/root/go/src/github.com/Azure/AgentBaker/hack/tools/bin go get github.com/go-bindata/go-bindata/...@v3.1.2
go: finding github.com/go-bindata/go-bindata v3.1.2
make[1]: Leaving directory '/root/go/src/github.com/Azure/AgentBaker/hack/tools'
-mod=vendor
go-bindata 3.1.2 (Go runtime go1.13.4).
Copyright (c) 2010-2013, Jim Teeuwen.
(pushd parts && \
../hack/tools/bin/go-bindata --nometadata --nocompress -pkg templates -o ../pkg/templates/templates_generated.go ./... && \
popd \
)
~/go/src/github.com/Azure/AgentBaker/parts ~/go/src/github.com/Azure/AgentBaker
~/go/src/github.com/Azure/AgentBaker
go build -mod=vendor -ldflags '-s -X main.version=8af2a3812ad54e264beb92243320e3456f66ab3c -X github.com/Azure/agentbaker/pkg/test.JUnitOutDir=/root/go/src/github.com/Azure/AgentBaker/test/junit -X github.com/Azure/agentbaker/cmd.BuildSHA=8af2a38 -X github.com/Azure/agentbaker/cmd.GitTreeState=clean -X github.com/Azure/agentbaker/cmd.BuildTag=canary' -o /root/go/src/github.com/Azure/AgentBaker/bin/baker
# github.com/Azure/agentbaker/cmd
cmd/generate.go:237:56: not enough arguments in call to templateGenerator.GetNodeBootstrappingCmd
        have (*api.ContainerService, *api.AgentPoolProfile, string, string, string, string)
        want (*api.ContainerService, *api.AgentPoolProfile, string, string, string, string, bool, bool)
Makefile:98: recipe for target 'go-build' failed
make: *** [go-build] Error 2

```

File: parts/linux/cloud-init/artifacts/cse_install.sh [hunk 1]
```diff
@@ -27,7 +27,7 @@ installDeps() {
     aptmarkWALinuxAgent hold
     apt_get_update || exit $ERR_APT_UPDATE_TIMEOUT
     apt_get_dist_upgrade || exit $ERR_APT_DIST_UPGRADE_TIMEOUT
-    for apt_package in apache2-utils apt-transport-https blobfuse ca-certificates ceph-common cgroup-lite cifs-utils conntrack cracklib-runtime ebtables ethtool fuse git glusterfs-client htop iftop init-system-helpers iotop iproute2 ipset iptables jq libpam-pwquality libpwquality-tools mount nfs-common pigz socat sysstat traceroute util-linux xz-utils zip; do
+    for apt_package in apache2-utils apt-transport-https blobfuse=1.1.1 ca-certificates ceph-common cgroup-lite cifs-utils conntrack cracklib-runtime ebtables ethtool fuse git glusterfs-client htop iftop init-system-helpers iotop iproute2 ipset iptables jq libpam-pwquality libpwquality-tools mount nfs-common pigz socat sysstat traceroute util-linux xz-utils zip; do
```
Comment (xuto2) [inline parts/linux/cloud-init/artifacts/cse_install.sh:30]: is there any fix that is needed in 1.2.3? I hope the downgrade will not introduce more bug?

Comment (andyzhangx) [reply parts/linux/cloud-init/artifacts/cse_install.sh:30]: we are on 1.1.1 for almost a year, it's quite stable.

**Pull request merged**